_11ty/plugins/markdown-plugins.js

```diff
@@ -33,11 +33,8 @@ module.exports = function syntaxHighlighting(eleventyConfig) {
             return slug(text.replace(/[<>()[\]{}]/gu, ""))
 
                 // non-ASCII characters are a pain to fix
```

Comment:
I think we can now replace this with a `// remove non-ASCII characters` comment.